folding-schemes/Cargo.toml

```diff
@@ -23,9 +23,13 @@ num-integer = "0.1"
 color-eyre = "=0.6.2"
 ark-bn254 = {version="0.4.0"}
 ark-groth16 = { version = "^0.4.0" }
+ark-noname = { git = "https://github.com/dmpierre/ark-noname", branch="feat/sonobe-integration" }
+noname = { git = "https://github.com/dmpierre/noname" }
+serde_json = "1.0.85"                                                                # to (de)serialize JSON
 
 # tmp imports for espresso's sumcheck
 espresso_subroutines = {git="https://github.com/EspressoSystems/hyperplonk", package="subroutines"}
+serde = "1.0.203"
```

Comment:
does this one fall under the `# tmp imports for espresso's sumcheck` imports section?
since we're at it, if you can also update the old comment from `imports` to `import` (singular), thanks ^^

Reply:
[Corrected](https://github.com/privacy-scaling-explorations/sonobe/blob/d2db27d434dd70c5078cbd1738a9e8e6e0577c4d/folding-schemes/Cargo.toml#L33)